DOCS/man/options.rst

```diff
@@ -1300,13 +1302,13 @@ Video
         When testing, you should start by using ``hwdec=auto-safe`` as it will
         limit itself to choosing from hwdecs that are actively supported by the
         development team. If that doesn't result in working hardware decoding,
-        you can try ``hwdec=auto`` to have it attempt to load every possible
-        hwdec, but if ``auto-safe`` didn't work, you will probably need to know
-        exactly which hwdec matches your hardware and read up on that entry
-        below.
+        you can try ``hwdec=auto-unsafe`` to have it attempt to load every
+        possible hwdec, but if ``auto-safe`` didn't work, you will probably need
+        to know exactly which hwdec matches your hardware and read up on that
+        entry below.
 
-        If ``auto-safe`` or ``auto`` produced the desired results, we recommend
-        just sticking with that and only setting a specific hwdec in your config
+        If ``auto-safe`` produced the desired results, we recommend just
+        sticking with that and only setting a specific hwdec in your config
```

Comment:
I don't understand this paragraph. Wouldn't it be better for users to select specific api that works best for them? I understand `auto-safe` is easy one liner, but is this always the best?

Reply:
Yeah that was already there before. I wasn't sure about changing it or not.